src/components/clientSettings/clientSettings.tsx

```diff
@@ -0,0 +1,190 @@
+		setQuantityValue(event.target.value);
+	};
+
+	const handleChangeActive = (e: FormEvent<HTMLDivElement>) => {
+		const target = e.target as HTMLDivElement;
+		setActive({ [target.id]: target.id });
+	};
+	const cancelHandler = () => {
+		setWebsiteValue('');
+		setIndustryValue(initialCountry);
+		setQuantityValue('');
+		reset(initial);
+	};
+
+	const onSubmit = (data: Data) => {
+		const NewData = {
+			...data,
+			website,
+			industry,
+			quantity,
+		};
+	};
+
+	return (
+		<Container>
+			<MinColumn>
+				<ButtonBlock>
+					<Button className={active?.settings ? 'defaultActive' : ''} onClick={handleChangeActive}>
+						<span id={settings}>{`${t('ClientSettings.settings')}`}</span>
+					</Button>
+					<Button className={active?.password ? 'defaultActive' : ''} onClick={handleChangeActive}>
+						<span id={password}>{`${t('ClientSettings.password')}`}</span>
+					</Button>
+				</ButtonBlock>
+			</MinColumn>
+			<MaxColumn>
+				{active?.settings === settings && (
+					<form onSubmit={handleSubmit(onSubmit)}>
+						<MainTitle>
+							{`${t('ClientSettings.title')}`}
+							<button>
+								<Img src={editIcon}></Img>
+							</button>
+						</MainTitle>
+						<div>
+							<Column>
+								<Title>{`${t('ClientSettings.name')}`}</Title>
+								<Input
+									type="text"
+									{...register('name')}
+									className={`form-control ${errors.name ? 'is-invalid' : ''}`}
+								/>
+							</Column>
+							<Column>
+								<Title>{`${t('ClientSettings.email')}`}</Title>
+								<Input type="text" />
+							</Column>
+							{errors.name && <P>{errors.name?.message}</P>}
+							<Div>
+								<Title>{`${t('ClientSettings.country')}`}</Title>
+								<Controller
+									name="country"
+									control={control}
+									render={({ field }) => {
+										return (
+											<Select
+												{...field}
+												options={options}
+												className={`${errors.country ? 'is-invalid' : ''}`}
+											/>
+										);
+									}}
+								/>
+								{errors.country && <P>{errors.country.value?.message}</P>}
+							</Div>
+						</div>
+						<Div>
+							<Title>{`${t('ClientSettings.website')}`}</Title>
+							<Input type="text" onChange={websiteChangeHandler} value={website} />
+						</Div>
+						<Column>
+							<Title>{`${t('ClientSettings.industry')}`}</Title>
+							<Select
+								options={selection}
+								onChange={choice => setIndustryValue(choice as ICountry)}
+								value={industry}
+							/>
+						</Column>
+						<Column>
+							<div>
+								<Title>{`${t('ClientSettings.quantity')}`}</Title>
+								<CheckLabel>
+									{checkList.map((item: string, index: number) => (
+										<div key={index}>
+											<input
+												value={item}
+												type="radio"
+												name="gender"
+												onChange={quantityHandleChange}
+												checked={item === quantity}
+											/>
```

Comment:
you have inputs in the project, why dont use exist one?

Reply:
reused